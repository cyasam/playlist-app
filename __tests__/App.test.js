import React from 'react'
import { shallow } from 'enzyme'
import App from '../src/scripts/app'

describe('App Component', () => {
  const wrapper = shallow(<App />)

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('has Header Component', () => {
    expect(wrapper.find('Header').exists()).toBe(true)
  })

  it('has Container Component', () => {
    expect(wrapper.find('Container').exists()).toBe(true)
  })
})