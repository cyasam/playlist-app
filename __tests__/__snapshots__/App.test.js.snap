// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders properly 1`] = `
<div
  className="wrapper"
>
  <Connect(Header) />
  <Container />
</div>
`;
